awx/main/tasks.py

```diff
@@ -2252,7 +2252,7 @@ def make_local_copy(project_path, destination_folder, scm_type, scm_revision):
             git_repo = git.Repo(project_path)
             if not os.path.exists(destination_folder):
                 os.mkdir(destination_folder, stat.S_IREAD | stat.S_IWRITE | stat.S_IEXEC)
-            tmp_branch_name = 'awx_internal/{}'.format(uuid4())
+            tmp_branch_name = 'awx_internal_{}'.format(uuid4())
```

Comment:
:thinking: 
I do not doubt that this fixes the problem, but given that we already include a uuid in the branch name, I'm wondering why removing the forward slash prevents a duplicate file exception ?  

Is it because we're always creating an `awx_internal` _directory_ (because of the slash) and so the uuids don't matter?

Reply:
yeah I'm guessing that's probably it:

```
FileExistsError: [Errno 17] File exists: '/var/lib/awx/projects/_71__project_aspectmatter/.git/refs/heads/awx_internal' <----------------------------------------- 
```

Reply:
It's how git works, in that it nests these refs under directories when there are slashes. The race condition is where the library we use doesn't account for multi-processing, and tests for existence and creates the directory with a time gap in-between.